src/objects/jira_base.py

```diff
@@ -148,6 +147,23 @@ def create_issue(
 
         return issue
 
+    def add_attachment_to_issue(self, issue: Issue, attachment_path: str) -> None:
+        """
+        Add and upload the attachment from attachment_path to the Jira Issue object provided.
+
+        Args:
+            issue (Issue): The Jira issue the attachment should be added to.
+            attachment_path (str): The path of the file to upload as an attachment to the issue.
+
+        Returns:
```

Comment:
No need to specify return if it's None

Reply:
> No need to specify return if it's None

Please see [previous discussion](https://github.com/RedHatQE/firewatch/pull/145/files#r1471572776).